system/common.xml

```diff
@@ -177,6 +237,13 @@
 				<echo message="${SYSTEMTEST_ROOT}/openjdk-systemtest exists, skip cloning" />
 			</else>
 		</if>
+		<if>
+			<isset property="isZOS" />
+			<then>
+				<echo message="Clone SVTTestRepo freshly on z/OS" />
+				<antcall target="clone_svttestrepo" inheritall="true" />
```

Comment:
This change would cause system/common.xml to clone and compile SVTTestRepo every time we are on z/OS, regardless of SDK version. But SVTTestRepo contains test materials that do not compile on OpenJ9, whereas system/common.xml is used commonly across all SDK versions and types.